package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "test": "node_modules/.bin/_mocha  $(find . -name '*.spec.js')",
-    "test:lint": "node_modules/eslint/bin/eslint.js ./  || exit 0",
+    "test:lint": "eslint .",
```

Comment:
`node_modules/.bin` is put on `$PATH` by npm automatically, so it can be called like this